web_external/models/DatasetModel.js

```diff
@@ -48,14 +48,42 @@ const DatasetModel = MinervaModel.extend({
         if (this.getDatasetType().match(/(geo)?json/)) {
             var geoData = geojsonUtil.normalize(this.get('geoData'));
             var visProperties = this.getMinervaMetadata().visProperties;
+            // Apply max and min value as styling color clamp initial value
             if (visProperties && visProperties.polygon && visProperties.polygon.fillColorKey && geoData.summary[visProperties.polygon.fillColorKey]) {
                 visProperties.polygon.maxClamp = geoData.summary[visProperties.polygon.fillColorKey].max;
                 visProperties.polygon.minClamp = geoData.summary[visProperties.polygon.fillColorKey].min;
             }
+            if (visProperties && visProperties.polygon && !visProperties.polygon.fillColorKey) {
+                var fillColorProperties = this.tryGetDefaultFillColor(geoData.summary);
+                if (fillColorProperties) {
+                    this.getMinervaMetadata().visProperties.polygon = Object.assign(this.getMinervaMetadata().visProperties.polygon, fillColorProperties);
+                }
+            }
             this.set('geoData', geoData);
         }
     },
 
+    tryGetDefaultFillColor(summary) {
+        var numberFields = _.pairs(summary).filter((pair) => {
+            return !pair[1].values;
```

Comment:
Although this may have `values`, that isn't a guarantee that it has `min` and `max`.  I think a better check would be
`return !pair[1].values && isFinite(pair[1].min) && isFinite(pair[1].max) && pair[1].min !== pair[1].max;`